pkg/directory/directory_unix.go

```diff
@@ -12,6 +12,10 @@ import (
 func Size(dir string) (size int64, err error) {
 	data := make(map[uint64]struct{})
 	err = filepath.Walk(dir, func(d string, fileInfo os.FileInfo, e error) error {
+		if e != nil {
```

Comment:
Can we use `err` please as a variable name. Also probably makes sense to ignore `os.IsNotExist`